### PR TITLE
Add Contributor state to API 

### PR DESCRIPTION
**Pull request by gu3st**
Created contributors file for OpenDota contributors to flag their profile if desired.  Added it to the API response for /players/:id.

### CODE REVIEW

File: CONTRIBUTORS.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*
```
Comment (howardchung) [inline CONTRIBUTORS.js]: might be easier to just make this a JSON file

File: package-lock.json [hunk 2]
```diff
@@ -10,7 +10,7 @@
       "integrity": "sha512-OfC2uemaknXr87bdLUkWog7nYuliM9Ij5HUcajsVcMCpQrcLmtxRbVFTIqmcSkSeYRBFBRxs2FiUqFJDLdiebA==",
```
Comment (howardchung) [inline package-lock.json]: don't think package-lock needs to be changed

File: routes/spec.js [hunk 3]
```diff
@@ -1017,6 +1018,11 @@ Please keep request rate to approximately 1/s.
                       description: 'loccountrycode',
                       type: 'string',
                     },
+                    is_contributor: {
+                      description: 'Has the user contributed to the development of OpenDota',
```
Comment (howardchung) [inline routes/spec.js]: I think better to start with "Boolean indicating if the user has contributed..."

File: routes/spec.js [hunk 4]
```diff
@@ -1028,7 +1034,12 @@ Please keep request rate to approximately 1/s.
           const accountId = Number(req.params.account_id);
           async.parallel({
             profile(cb) {
-              queries.getPlayer(db, accountId, cb);
+              queries.getPlayer(db, accountId, (err, playerData) => {
+                if (!err && playerData !== undefined) {
```
Comment (howardchung) [inline routes/spec.js]: think you can just check if (playerData)

File: CONTRIBUTORS.js (hunk 1 above)
Comment (howardchung) [reply CONTRIBUTORS.js]: Also add:
88367253 (me)
102344608 (@albertcui)

Comment (gu3st) [reply CONTRIBUTORS.js]: The only reason I made it a .js was so I could include a comment to explain the file. If JSON supported comments I would have done JSON.

Comment (howardchung) [reply CONTRIBUTORS.js]: ok, that's fine :)

**Pull request merged**